docs/projects/ad7606x_fmcz/index.rst

```diff
@@ -0,0 +1,433 @@
+-  :ref:`SPI_ENGINE <spi_engine>`
+
+.. list-table::
+   :widths: 30 35 35
+   :header-rows: 1
+
+   * - IP name
+     - Source code link
+     - Documentation link
+   * - SYNC_BITS
+     - :git-hdl:`library/util_cdc/sync_bits.v <main:library/util_cdc/sync_bits.v>` **
+     - ---
+   * - AD_EDGE_DETECT
+     - :git-hdl:`library/common/ad_edge_detect.v <main:library/common/ad_edge_detect.v>`
+     - ---
+   * - AXI_AD7606x
+     - :git-hdl:`library/axi_ad7606x <main:library/axi_ad7606x>` *
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_ad7606x>`
+   * - AXI_CLKGEN
+     - :git-hdl:`library/axi_dmac <main:library/axi_clkgen>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_clkgen>`
+   * - AXI_DMAC
+     - :git-hdl:`library/axi_dmac <master:library/axi_dmac>`
+     - :ref:`here <axi_dmac>`
+   * - AXI_HDMI_TX
+     - :git-hdl:`library/axi_hdmi_tx <main:library/axi_hdmi_tx>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_hdmi_tx>`
+   * - AXI_I2S_ADI
+     - :git-hdl:`library/axi_i2s_adi <main:library/axi_i2s_adi>`
+     - ---
+   * - AXI_PWM_GEN
+     - :git-hdl:`library/axi_pwm_gen <main:library/axi_pwm_gen>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_pwm_gen>`
+   * - AXI_SPDIF_TX
+     - :git-hdl:`library/axi_spdif_tx <main:library/axi_spdif_tx>`
+     - ---
+   * - AXI_SYSID
+     - :git-hdl:`library/axi_sysid <main:library/axi_sysid>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_sysid>`
+   * - SYSID_ROM
+     - :git-hdl:`library/sysid_rom <main:library/sysid_rom>`
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/axi_sysid>`
+   * - UTIL_I2C-MIXER
+     - :git-hdl:`library/util_i2c_mixer <main:library/util_i2c_mixer>`
+     - ---
+   * - UTIL_CPACK2
+     - :git-hdl:`library/util_pack/util_cpack2 <main:library/util_pack/util_cpack2>` *
+     - :dokuwiki:`[Wiki] <resources/fpga/docs/util_cpack>`
+   * - AXI_SPI_ENGINE
+     - :git-hdl:`library/spi_engine/axi_spi_engine <main:library/spi_engine/axi_spi_engine>`  **
+     - :ref:`here <spi_engine axi>`
+   * - SPI_ENGINE_EXECUTION
+     - :git-hdl:`library/spi_engine/spi_engine_execution <main:library/spi_engine/spi_engine_execution>` **
+     - :ref:`here <spi_engine execution>`
+   * - SPI_ENGINE_INTERCONNECT
+     - :git-hdl:`library/spi_engine/spi_engine_interconnect <main:library/spi_engine/spi_engine_interconnect>` **
+     - :ref:`here <spi_engine interconnect>`
+   * - SPI_ENGINE_OFFLOAD
+     - :git-hdl:`library/spi_engine/spi_engine_offload <main:library/spi_engine/spi_engine_offload>` **
+     - :ref:`here <spi_engine offload>`
+
+.. admonition:: Legend
+   :class: note
+
+   -   ``*`` instantiated only for INTF=0 (parallel interface)
+   -   ``**`` instantiated only for INTF=1 (serial interface)
+
+Software related
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  :git-no-os:`AD7606 - No-OS Driver <drivers/adc/ad7606>`
+-  :dokuwiki:`AD7606 - No-OS Driver <resources/tools-software/uc-drivers/ad7606>`
```

Comment:
Specify that this link is to a documentation, otherwise they both look the same when generating the documentation

Reply:
Done.